src/fuzzer/mem_pool.cpp

```diff
@@ -12,6 +12,7 @@
 #include <utility>
 
 #include <stdlib.h>
+#include <cstdlib>
```

Comment:
Shouldn't it be okay to remove `#include <stdlib.h>` if you include `cstdlib`? If both need to stay for some sort of legacy workaround, please leave a comment explaining why.

Reply:
I don't know, I didn't check further. 

If I remove stdlib.h does part of the code have to be rewritten to conform to cstdlib? I'm not at my ease in c/c++, sorry.

Reply:
Usually, `cstdlib` should be enough. Could you try to remove `stdlib.h` and see if it still compiles on the old platforms?

On my macOS (11.6.1) it works just fine without `stdlib.h`.

Reply:
I can only check on 10.13. since it builds on macports with this patch on 10.11 it will be difficult to get such a test (just to test that it will builds) accepted in macports' production servers.

Reply:
It builds fine on 10.13. I will re-commit.